src/Ledger/Gov.lagda

```diff
@@ -3,14 +3,15 @@
 \begin{code}[hide]
 {-# OPTIONS --safe #-}
 
-open import Axiom.Set.Properties using (∃?-sublist-⇔)
 open import Ledger.Prelude hiding (any?; Any; all?; All; Rel; lookup; ∈-filter)
```

Comment:
Why does this import appear before the module declaration?  Isn't this inconsistent with the rest of the library?

Reply:
Ah, I missed that!